Formula/postgresql@11.rb

```diff
@@ -79,8 +79,10 @@ def post_install
 
   def caveats
     <<~EOS
-      To migrate existing data from a previous major version of PostgreSQL run:
-        brew postgresql-upgrade-database
```

Comment:
Please don't remove this.

Reply:
Sorry, that was unintentional. Fixed in a new commit (let me know if you want me to squash all these commits before you merge)